inttest/openebs/openebs_test.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2024 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openebs
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/k0sproject/k0s/inttest/common"
+	"github.com/k0sproject/k0s/pkg/kubernetes/watch"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	helmv1beta1 "github.com/k0sproject/k0s/pkg/apis/helm/v1beta1"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type OpenEBSSuite struct {
+	common.BootlooseSuite
+}
+
+func (s *OpenEBSSuite) TestK0sGetsUp() {
+	ctx := s.Context()
+	s.T().Run("Both storage and helm extensions enabled", func(t *testing.T) {
+		s.PutFile(s.ControllerNode(0), "/tmp/k0s.yaml", k0sConfigWithBoth)
+		s.Require().NoError(s.InitController(0, "--config=/tmp/k0s.yaml", "--disable-components=konnectivity-server,metrics-server"))
+		s.Require().NoError(s.RunWorkers())
+
+		kc, err := s.KubeClient("controller0", "")
+		s.Require().NoError(err)
+
+		err = s.WaitForNodeReady("worker0", kc)
+		s.NoError(err)
+
+		err = s.WaitForNodeReady("worker1", kc)
+		s.NoError(err)
```

Comment:
For consistency with other tests:

```suggestion
		kc, err := s.KubeClient(s.ControllerNode(0), "")
		s.Require().NoError(err)

	for idx := 0; idx < s.BootlooseSuite.WorkerCount; idx++ {
		err := s.WaitForNodeReady(s.WorkerNode(idx), kc)
		s.Require().NoError(err)
	}
```

Reply:
Since there is only one node now I think 

```
		kc, err := s.KubeClient(s.ControllerNode(0), "")
		s.Require().NoError(err)

		s.Require().NoError(s.WaitForNodeReady(s.WorkerNode(0), kc))
```

Would you be ok with that?